roles/packages/tasks/main.yml

```diff
@@ -4,6 +4,10 @@
 #
 # updates apt, installs packages used by other installation/configuration tasks
 #
+- name: install aptitude
+  become: yes
+  apt: name=aptitude state=installed
```

Comment:
Can you add a comment about why this is pulled out separately instead of in the list of dev tools packages below.  Otherwise, I'm afraid I or someone else later would see it and refactor it into that list because it looks like just another package install.